src/Core/Domain/CartRule/QueryResult/CartRuleForEditing.php

```diff
@@ -129,4 +131,124 @@ public function getDateUpd(): ?DateTime
     {
         return $this->dateUpd;
     }
+
+    /**
+     * @return string
+     */
+    public function getDescription(): string
+    {
+        return $this->information->getDescription();
+    }
+
+    /**
+     * @return string
+     */
+    public function getCode(): string
+    {
+        return $this->information->getCode();
+    }
+
+    /**
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return $this->information->getPriority();
+    }
+
+    /**
+     * @return bool
+     */
+    public function getAllowPartialUse(): bool
+    {
+        return $this->information->isPartialUse();
+    }
+
+    /**
+     * @return bool
+     */
+    public function getActive(): bool
+    {
+        return $this->information->isEnabled();
+    }
+
+    /**
+     * @return array
+     */
+    public function getLocalizedNames(): array
+    {
+        return $this->information->getLocalizedNames();
+    }
+
+    /**
+     * @return bool
+     */
+    public function getHighlightInCart(): bool
+    {
+        return $this->information->isHighlight();
+    }
+
+    public function getMinimumAmount(): CartRuleMinimumAmount
+    {
+        return new CartRuleMinimumAmount(
+            (float) (string) $this->conditions->getMinimum()->getAmount(),
```

Comment:
I don't know if we can use `DecimalNumberNormalizer`here?

Reply:
Definitely ! That would be great if this is well designed, and can be reused for other endpoints